### ISSUE TITLE
Task and Tag handling collection

### ISSUE THREAD

**Issue by Jagdfalke** [super-productivity/super-productivity#3111]
This issue lists all feature requests regarding the handling of tasks and tags.

**Features:**

### Copy or Template Concepts
- [ ] #2720
- [ ] #3025
- [x] #2963
- [x] #2931
- [x] #2694

### Tags
- [x] #2946
- [x] #2895
- [ ] Allow sorting by multiple tags #2920
- [ ] #3041

### Shortcuts or Syntax
- [x] #2983
- [x] #3001 
- [x] #2692
- [x] #2689

### Subtasks in Integrations
- [ ] #2876
- [ ] #150

### Repeated Tasks
- [ ] #2471
- [x] #2752
- [x] #2938
- [x] #427

### Task Handling
- [x] #2936
- [ ] #3137
- [ ] #2934
- [x] #2933
- [x] #2926
- [x] #2829
- [ ] #2778
- [ ] #1639
- [x] #2775
- [ ] #2657
- [x] #992
- [ ] #3072
- [ ] #2830
- [x] #810
- [x] #1828
- [x] #2932

**Comment by bytrangle** [comment]
What a thorough collection. However, without further context, aspiring contributors may not know where to start. Examples of context: difficulty level, importance of the issue etc.

**Comment by Jagdfalke** [comment]
Yes, true. At the moment, this collection serves both Johannes and myself as an overview over what users want, what similar needs they have and in what areas Super Productivity can improve on. 

At this point it is rather difficult to judge which issues would be most suited for new contributors.

**Comment by bytrangle** [comment]
I do have some issues in mind. They are already in the collection :). However, I do want to make sure that the contributions I made are really something that the maintainers want and they are in line with their vision for SP.

**Comment by johannesjo** [comment]
> I do have some issues in mind. They are already in the collection :). However, I do want to make sure that the contributions I made are really something that the maintainers want and they are in line with their vision for SP.

What 2 features or improvement do you miss the most yourself? :)

**Comment by bytrangle** [comment]
> What 2 features or improvement do you miss the most yourself? :)

Personally, I use tags a lot, but I don't use the "#" shortcut because I'm lazy about typing the whole word, and sometimes I can't remember the the correct tag name. Did I use the word "react" or "react.js"? Did I use "watercolor" or "watercolour"?

Instead of using the shortcut,  after creating a task, I right click and choose "edit tags".

So I guess I may tackle the shortcut autocomplete first. This feature can also be used for adding projects.

**Comment by johannesjo** [comment]
I agree! The autocomplete for tags would be super useful!!!

**Comment by EternityMar22** [comment]
> I agree! The autocomplete for tags would be super useful!!!
Please make sure you focus on fixing the synchronization issue, which is causing a great deal of trouble for basic use. See "https://github.com/johannesjo/super-productivity/issues/3288" for details. Next is the ios version, even the unsigned version is acceptable.

**Comment by mighty-services** [comment]
I'd like to add field for specific Start and Stop Times while pressing the "play" button or some other way to input specific timeslots.
